guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>This <s>underlying</s> transformer model is trained</p><p> </p><p>The model assumes by default that only one of the candidate labels is true<s>, returning</s> and returns a list of scores for each label which add up to 1. Visit <a href="https://huggingface.co/models?search=nli" target="_blank">this</a> link to learn more about the available models for <strong>zero-shot-classification</strong> task. To get a list of supported transformer backbones for this model use the below command.</p><p> </p><p>Hyperlink <strong>this link</strong> together</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Made the necessary changes</p>